extensions/azure-extensions/src/main/java/io/druid/storage/azure/AzureLoadSpec.java

```diff
@@ -31,24 +31,24 @@
 public class AzureLoadSpec implements LoadSpec
 {
   @JsonProperty
-  private final String storageDir;
+  private final String blobPath;
 
   private final AzureDataSegmentPuller puller;
 
   @JsonCreator
   public AzureLoadSpec(
-      @JacksonInject AzureDataSegmentPuller puller,
-      @JsonProperty("storageDir") String storageDir
```

Comment:
Comparing S3 to Azure, it seems like the concepts of "bucket" and "key" in S3, map to "container" and "blob" in Azure.
It seems strange that we would configure the container globally for Azure deep storage, given that bucket is configured on a per segment basis for S3 deep-storage.

Is there any reason we could not have Azure loadSpec be as close as possible to the S3 loadSpec and simple have "container" and "blob" as its two parameters?